packages/SwingSet/src/kernel/vat-warehouse.js

```diff
@@ -140,16 +146,16 @@ export function makeSyscallSimulator(
     // slog entries have no kernel-translated kso/ksr
     const finish = kernelSlog.syscall(vatID, undefined, vso);
     const expected = syscallsExpected[syscallsMade.length];
-    syscallsMade.push(vso);
+    const idx = syscallsMade.push(vso) - 1;
     if (!expected) {
       syscallStatus.push('extra');
-      const error = Error(`anachrophobia in ${vatID}: extra syscall`);
+      const error = Error(`${context}: extra syscall at index ${idx}`);
```

Comment:
I've been using `sc123` locally when marking syscall numbers (like b123 for block heights, c123 for cranks, d123 for deliveryNums), maybe this could say `at index sc${idx}` to reinforce the fact that we're talking about a `syscallNum` which could be correlated with one in a slogfile.